libcamera/examples/read_control.rs

```diff
@@ -0,0 +1,38 @@
+use libcamera::{camera_manager::CameraManager, controls::ControlId, logging::LoggingLevel};
+
+fn main() {
+    let mgr = CameraManager::new().unwrap();
+
+    mgr.log_set_level("Camera", LoggingLevel::Error);
+
+    let cameras = mgr.cameras();
+
+    //Grab the first camera, if one exists
+    for cam in cameras.iter().take(1) {
+        println!("ID: {}", cam.id());
+
+        //Read the first ControlInfo
+        for (id, control_info) in cam.controls().into_iter().take(1) {
```

Comment:
Is there a reason to only print the first one? I would think listing them all out would make for a more helpful and interesting demo/example?

Reply:
The reason this raised my eye was that different versions of libcamera resulted in this giving different values as the c++ was giving different order. I think for a example just taking the first works for people to get the idea of how to use the lib :)